src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -294,6 +294,23 @@ public void scheduleEmailsForSending(List<EmailWrapper> emails) {
         }
     }
     
+    /**
+     * Schedules production of searchable documents
+     * Task queue handles this so that instructor account creation depends on less database calls
+     * 
+     * @param courseId the course ID of the feedback session
+     * @param feedbackSessionName the name of the feedback session
+     * @param enrollmentList the list of enrollment details
```

Comment:
Looks like this is copied from somewhere and forgot to be modified :p

- "the course ID of the feedback session" is inaccurate: there is no "feedback session" to be talked about.
- The last two params do not exist.